src/extension/app/store/app.js

```diff
@@ -210,6 +210,8 @@ export class AppStore {
     });
 
     this.showView();
+
+    document.dispatchEvent(new CustomEvent(EXTERNAL_EVENTS.SIDEKICK_READY));
```

Comment:
If we fire `SIDEKICK_READY` here, I think we should remove the `CONTEXT_LOADED` one fired above. At least I don't see whom it would benefit, and we don't seem to be usig it internally either.

Reply:
This would essentially become the new context loaded event.